### ISSUE TITLE
Font smoothing issue on non-retina screen with desktop safari

### ISSUE THREAD

**Issue by muratcorlu** [jdan/98.css#13]
If you open the page in an external(non-retina) screen in latest Safari on MacOS Catalina, font smoothing doesn't render properly. Here is a screenshot:

![image](https://user-images.githubusercontent.com/127687/79964924-ab734400-848b-11ea-91de-6c656d339bd1.png)

**Comment by grstreten** [comment]
Echo of this on an  Thunderbolt Display.

**Comment by jdan** [comment]
Yikes! Sorry about this. Wondering if the `letter-spacing` I just removed will fix this, or if I need to make sure not to remove `-webkit-font-smoothing` for this scenario.

**Comment by muratcorlu** [comment]
I also played with it but couldn't find an easy solution. I'll also try playing with `letter-spacing`.

**Comment by muratcorlu** [comment]
Disabling `letter-spacing` doesn't help.

**Comment by jdan** [comment]
Hey @muratcorlu!

We disabled font-smoothing when we brought the new font in. Wanna take a look at https://jdan.github.io/98.css (hard refresh with command+shift+R) and let me know how it looks?

**Comment by grstreten** [comment]
Whilst I'm yet to check on a non-retina display, rendering in Safari is looking a little funky now. Apologies if this needs a separate issue, happy to raise it in one. Verified it's not occurring in Chrome.

<img width="337" alt="Screenshot 2020-04-23 at 22 47 13" src="https://user-images.githubusercontent.com/2766825/80153004-a3fb8a00-85b4-11ea-9421-df7f0c4cc739.png">

**Comment by muratcorlu** [comment]
Hi @jdan 

Here is the latest situation:

![image](https://user-images.githubusercontent.com/127687/80185504-b9040780-860c-11ea-9540-ab79028e43cc.png)

**Comment by jdan** [comment]
Ran into a fun one: https://github.com/jdan/98.css/issues/68#issuecomment-620060479

I know nothing about web fonts :)

**Comment by jdan** [comment]
It's almost as if the space ruins it!

Hey @muratcorlu if you modify the "My First Program" header into "MyFirstProgram" does it go away?

**Comment by muratcorlu** [comment]
@jdan yes, without spaces it looks good.

![image](https://user-images.githubusercontent.com/127687/80483648-bbd96200-8956-11ea-9bdf-93c064f92217.png)

**Comment by muratcorlu** [comment]
Same applies for dialog content text. Without space, it's ok. What is the cause for this? 🤯

![image](https://user-images.githubusercontent.com/127687/80483803-f8a55900-8956-11ea-8873-b5b083be3fca.png)

**Comment by dertuxmalwieder** [comment]
It also looks like that on a Retina MacBook Pro, so the non-Retina is not the issue here.

**Comment by DannyJJK** [comment]
I'm not sure if this is an issue any more, it looks good on Safari 14.1.1 (16611.2.7.1.4)
<img width="780" alt="imagen" src="https://user-images.githubusercontent.com/11057501/120550690-1a400200-c3ed-11eb-8a92-1edb5d839650.png">

**Comment by dertuxmalwieder** [comment]
The "dialog" seems to be fixed, but the tree and dropdown boxes still look strange on Retina (Safari 14 TP).

<img width="624" alt="Bildschirmfoto 2021-06-02 um 22 57 36" src="https://user-images.githubusercontent.com/125629/120551234-15338080-c3f6-11eb-91f7-51cd51a7de9c.png">

**Comment by muratcorlu** [comment]
Here is the current situation for me on Safari 14.1.1 and on an external (non-retina) display.

![image](https://user-images.githubusercontent.com/127687/120602140-05925700-c44b-11eb-8cc0-5e428bae2da3.png)
![image](https://user-images.githubusercontent.com/127687/120602169-0d51fb80-c44b-11eb-8b36-a8449d034bce.png)

It's way better. But still some letters in that thin font overlapping.

**Comment by rolandcrosby** [comment]
Yep, I'm using an old non-Retina MacBook Air (Safari 14.1), and I see the same kerning issues as @muratcorlu is seeing. I don't believe there are any missing pixels or mis-rendered individual characters like in @dertuxmalwieder's screenshot, though.

**Comment by dertuxmalwieder** [comment]
If it helps, both XP.css and 7.css don't have these problems for me.
